x-pack/plugins/observability_solution/infra/public/components/asset_details/context_providers.tsx

```diff
@@ -44,22 +29,20 @@ export const ContextProviders = ({
   } = props;
 
   return (
-    <RenderWithOptionalSearchSessionProvider renderMode={renderMode}>
```

Comment:
Please leave this context here, it is used to control and force new requests. As said above it will be changed to not rely on `searchSessionId`.